net/README.md

```diff
@@ -0,0 +1,58 @@
+## DIY Guide for Contiv Networking
+
+### Pre-requisites
+1. You need Ubuntu 15.04 or higher on each of your servers used for the Contiv cluster.
+
+2. If your servers are behind an http proxy (usually the case in many cisco labs...), you need
+   to do "export http_proxy=<proxy url>" and "export https_proxy=<proxy_url>" in your shell
+
+3. You will select and use on server to initate installation on all servers in the cluster.
+   Please refrain from running install from multiple servers. Instead, stick to the same server to initiate
+   installation.
+
+4. It is recommended that you enable passwordless SSH access from the selected server to all
+   the other servers in the cluster.
+   An example of how to set this up is here: http://www.linuxproblem.org/art_9.html
```

Comment:
Can you create this as a clickable link. For example use following:
. . . here: [http://www.linuxproblem.org/art_9.html]